ci/docker.build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e -u
+
+cd /core/
+dub build -c static --compiler=ldc2
+
+# remove dub folder generated with build
+# as it has root permissions from docker in the build folder
+rm -rf ./.dub/
```

Comment:
Why the `rm` inside of the docker image ? It doesn't make much sense to me.

Reply:
docker is run with a root user, the folder is generated inside the build folder it is connected to. Without this the remove outside would need root access, which it doesn't have.